Beyond08-SOW.md

```diff
@@ -0,0 +1,485 @@
+you should take the time to think about how you perceive your own performance,
+how is that different from how the mentors see it and why. Take the opportunity
+to learn how to improve!
+
+### Demo 3 (January 1st 2023): Control logic and UI
+
+The purpose of this demo is to review your ability to implement the application
+workflow and UI logic and implement automated tests for it.
+
+**Technical Requirements**
+
+1. Complete the implementation of one or more features of your application,
+   including UI and backend logic. The features you implement should allow you
+   to demonstrate your ability to:
+
+    1. Query data from your database and present it to your users
+    2. Use forms to gather user input
+    3. Store or modify information in the database as a response to user input
+
+2. Implement automated tests for the features you've completed. The tests 
+   should cover all aspects of the feature's functionality including failure 
+   scenarios, handling of invalid data and unauthorized access scenarios. 
+
+**Demo contents**
+
+Your demo presentation should include the following elements:
+
+1. One or more video segments showing how the features you've implemented look
+   and work from a user's point of view.
+2. An explanation of how you've tested the features you've developed. You should
+   include the following details:
+
+    1. Which manual tests were carried out (If any). How does one decide if a 
+       test was successful?
+    2. Which automated tests were written
+    3. Are failure scenarios being tested? In which way?
+    4. What aspects of the features you've written are not being tested?
+3. A demonstration of the application tests being run in GitHub actions.
+
+### Demo 4 (January 22nd 2023): Final Project Demo
+
+The purpose of this demo is to present your entire development process and 
+should include:
+
+1. Project background - Explain in general the motivation and the requirements.
+2. Technical ideas and architecture - Explain test high-level technical 
+   ideas behind your project, please try to present concepts graphically, 
+   possibly by using ERD and DFD charts.
+4. Demo - A video that presents your full system (UI, DB, APIs, CI)
+5. Development process - share the process you made
+6. Development challenges
+7. What did you learn
+
+## Division into project groups (till November 6th)
+
+After the first 2 weeks, you'll be divided into teams - each team with 4-6 students.
+
+Each team will have the following mentors:
+
+* Team 1: Nati Friedman, TBD 
+* Team 2: Kobi Hakimi, TBD 
+* Team 3: Omer Amsalem, TBD
+* Team 4: Avi Biton, TBD
+
+## Project ideas
+
+You can pick from the ideas below or think up your own.
+
+* Platform for freelancers to offer their services
+* P2P Lending service
+* Dating website
+* Drop shipping web store - selling other people's products on your website
+* Platform for finding part-time jobs for students
```

Comment:
Lets drop this suggestion, since its been done to death.

In fact, perhaps lets add a list of things done 3+ time in Beyond already and encourage people not to do them...

Reply:
Ack